packages/tooltip/stories.tsx

```diff
@@ -86,13 +84,12 @@ storiesOf('Tooltip Container/useTooltip', module)
 
       const { isVisible, getTooltipProps, getTriggerProps, openTooltip, closeTooltip } = useTooltip(
         {
-          tooltipRef,
```

Comment:
`tooltipRef` does not seem to be used by the `useTooltip` hook so this this line of code is removed.

I verified the Tooltips on Storybook is still working, and verified that all tests still pass ✅ , but it may still be worth while to manually verify that there are no regressions in the Storybook for tooltip examples.